### PR TITLE
Refactor Compass test prow jobs

### PR DESCRIPTION
**Pull request by StanislavStefanov**
**Description**

Changes proposed in this pull request:

- add new prow job for refactored compass e2e tests - https://github.com/kyma-incubator/compass/pull/1760
- remove old compass e2e tests jobs

### CODE REVIEW

File: prow/jobs/incubator/compass/tests/compass-e2e-tests/compass-e2e-tests.yaml [hunk 1]
```diff
@@ -43,7 +43,7 @@ presubmits: # runs on PRs
         - ^main$
         - ^hotfix-.*$
     
-    - name: pre-rel120-compass-tests-director
+    - name: pre-rel120-compass-tests
```
Comment (desislavaa) [inline prow/jobs/incubator/compass/tests/compass-e2e-tests/compass-e2e-tests.yaml:46]: no need for this, it's Kyma specific

File: prow/jobs/incubator/compass/tests/compass-e2e-tests/compass-e2e-tests.yaml [hunk 2]
```diff
@@ -53,7 +53,7 @@ presubmits: # runs on PRs
       branches:
         - release-1.20
     
-    - name: pre-rel119-compass-tests-director
+    - name: pre-rel119-compass-tests
```
Comment (desislavaa) [inline prow/jobs/incubator/compass/tests/compass-e2e-tests/compass-e2e-tests.yaml:56]: no need for this, it's Kyma specific

File: prow/jobs/incubator/compass/tests/compass-e2e-tests/compass-e2e-tests.yaml [hunk 3]
```diff
@@ -63,7 +63,7 @@ presubmits: # runs on PRs
       branches:
         - release-1.19
     
-    - name: pre-rel118-compass-tests-director
+    - name: pre-rel118-compass-tests
```
Comment (desislavaa) [inline prow/jobs/incubator/compass/tests/compass-e2e-tests/compass-e2e-tests.yaml:66]: no need for this, it's Kyma specific

File: prow/jobs/incubator/compass/tests/compass-e2e-tests/compass-e2e-tests.yaml [hunk 4]
```diff
@@ -88,7 +88,7 @@ postsubmits:
       - ^master$
       - ^main$
       
-    - name: post-rel120-compass-tests-director
+    - name: post-rel120-compass-tests
```
Comment (desislavaa) [inline prow/jobs/incubator/compass/tests/compass-e2e-tests/compass-e2e-tests.yaml:91]: no need for this, it's Kyma specific

File: prow/jobs/incubator/compass/tests/compass-e2e-tests/compass-e2e-tests.yaml [hunk 5]
```diff
@@ -99,7 +99,7 @@ postsubmits:
       branches:
         - release-1.20
     
-    - name: post-rel119-compass-tests-director
+    - name: post-rel119-compass-tests
```
Comment (desislavaa) [inline prow/jobs/incubator/compass/tests/compass-e2e-tests/compass-e2e-tests.yaml:102]: no need for this, it's Kyma specific

File: prow/jobs/incubator/compass/tests/compass-e2e-tests/compass-e2e-tests.yaml [hunk 6]
```diff
@@ -110,7 +110,7 @@ postsubmits:
       branches:
         - release-1.19
     
-    - name: post-rel118-compass-tests-director
+    - name: post-rel118-compass-tests
```
Comment (desislavaa) [inline prow/jobs/incubator/compass/tests/compass-e2e-tests/compass-e2e-tests.yaml:113]: no need for this, it's Kyma specific

**Review by desislavaa — COMMENTED** [review]
We can also cleanup the kyma-specific jobs we currelntly have

**Review by desislavaa — APPROVED** [review]

**Review by dekiel — APPROVED** [review]

**Pull request merged**